jetty/src/test/scala/org/http4s/server/jetty/Issue454.scala

```diff
@@ -44,7 +44,7 @@ object Issue454 {
   }
 
   val servlet = new AsyncHttp4sServlet[IO](
-    service = HttpRoutes.of[IO] {
+    service = HttpApp {
       case GET -> Root => Ok(insanelyHugeData)
     },
```

Comment:
Please try and avoid inexhaustive matches. This can be done with `httpRoutesToApp`, or by adding a catch-all clause below.

Reply:
use the `.orNotFound` syntax in as described below